aunt-leahs-app/src/components/header/header.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SubText from './SubText.jsx';
+import { Typography } from '@material-ui/core';
+import constants from '../../constants.jsx';
+import '../../styles.css';
+
+const headerText = constants.headerConstants.headerText;
+
+const Head = (props) => {
+    /*takes in 'page' as props. Refer to ./components/constants.jsx for props.page values */
+    return(
+        <div>
+            <Typography variant="h6">
+                <h1 className="headerMain header">{headerText.headOne}</h1>
+                <h2 className="headerMain header">{headerText.headTwo}</h2>
+                {(props.page!=null) 
+                    &&<SubText title={props.page}/>}
+            </Typography>
+        </div>)
+};
+
+Head.defaultProps={page:null};
```

Comment:
I don't think you need this. By default (without this line), props.page would be undefined because that's just how JavaScript handles object properties that don't exist.
You would need this if you use !== on line 16 though, I think

Reply:
Noted